### ISSUE TITLE
Transformers error 

### ISSUE THREAD

**Issue by CristianonCarvalho** [stanfordnlp/dspy#21]
The following error is occurring when trying to run the Notebook for testing with KNN.
AttributeError: module 'dsp' has no attribute 'SentenceTransformersVectorizer'
Can you help me?

**Comment by okhat** [comment]
CC: @stalkermustang is the expert on this!

**Comment by stalkermustang** [comment]
Hi @CristianonCarvalho, could you please provide an example code to reproduce? Also, have you updated the library/repo? This is relatively new feature, so probably it isn't in your source files.

**Comment by CristianonCarvalho** [comment]
Just write this for test:
**import dsp
dsp.settings.configure(vectorizer=dsp.NaiveGetFieldVectorizer())**

---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-2-57882acac513> in <module>
      1 import dsp
----> 2 dsp.settings.configure(vectorizer=dsp.NaiveGetFieldVectorizer())

AttributeError: module 'dsp' has no attribute 'NaiveGetFieldVectorizer'

**Comment by stalkermustang** [comment]
@CristianonCarvalho did you install dsp-ml library using pip? and just copied .ipynb/code from the notebook, I suppose?

**Comment by CristianonCarvalho** [comment]
yes

**Comment by stalkermustang** [comment]
@CristianonCarvalho it seems that pypi version of the package wasn't updated since 0.1.4 release (Feb 13 from [here](https://pypi.org/project/dsp-ml/#history)). The PR with KNN functionality was merged [on Feb 19](https://github.com/stanfordnlp/dsp/pull/12). 

You can manually update the library using pip, smth like `pip install git+https://github.com/stanfordnlp/dsp.git` should help. Please note that you should uninstall your local version first (the -U flag won't help: both versions have 0.1.4 tag, so pip won't spot any differences, I suppose)

tldr:
`pip uninstall dsp-ml`
`pip install git+https://github.com/stanfordnlp/dsp.git`

**Comment by stalkermustang** [comment]
Hi @CristianonCarvalho did this solve the issue?

**Comment by aber0016** [comment]
@stalkermustang I had the same issue as @CristianonCarvalho and I was able to resolve it with the advice you gave

**Comment by CristianonCarvalho** [comment]
Yes, resolved. Thks!!!
